baseband/gsb/base.py

```diff
@@ -223,24 +219,16 @@ class GSBStreamReader(GSBStreamBase, VLBIStreamReaderBase):
     # be solved with FileWriter/FileReader classes that handle timestamps and
     # multiple blocks, combining these into a frame?
 
-    def __init__(self, fh_ts, fh_raw, thread_ids=None, nchan=None, bps=None,
-                 complex_data=None, samples_per_frame=None, payloadsize=None,
-                 frames_per_second=None, sample_rate=None, squeeze=True):
+    def __init__(self, fh_ts, fh_raw, sample_rate, thread_ids=None, nchan=None,
+                 bps=None, complex_data=None, samples_per_frame=None,
+                 payloadsize=None, squeeze=True):
         header0 = fh_ts.read_timestamp()
-        if frames_per_second is None and sample_rate is None:
```

Comment:
I'm reconsidering. Let's just leave `samples_per_frame` at the end (and organize the other keywords to be in order as similar as possible to what is done for the other formats), and calculate properly below (or just raise an error if `None` for now).